visual-js/visual/src/cli/build.ts

```diff
@@ -56,9 +56,11 @@ const getBuildResults = async (
   api: VisualApi,
   { buildId, customId }: { buildId?: string; customId?: string },
 ): Promise<Record<string, number>> => {
-  const build = customId
-    ? await api.buildWithDiffsByCustomId(customId)
-    : await api.buildWithDiffs(buildId);
+  let build;
+
+  if (customId) build = await api.buildWithDiffsByCustomId(customId);
+  if (buildId) build = await api.buildWithDiffs(buildId);
```

Comment:
Shouldn't we invert those line to ensure `customId` takes precedence ?

Reply:
Updated this to perform more closely to the original code and added a test.